CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'FF.F.FFFFF\n======================================================================\nFAIL: test0 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 12, in test0\n    self.assertEqual(patched_source(input_0), original_source(input_0))\nAssertionError: Lists differ: [8092] != [8192]\n\nFirst differing element 0:\n8092\n8192\n\n- [8092]\n?   ^\n\n+ [8192]\n?   ^\n\n\n======================================================================\nFAIL: test1 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 18, in test1\n    self.assertEqual(patched_source(input_1), original_source(input_1))\nAssertionError: Lists differ: [1060634624] != [1073741824]\n\nFirst differing element 0:\n1060634624\n1073741824\n\n- [1060634624]\n+ [1073741824]\n\n======================================================================\nFAIL: test3 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 30, in test3\n    self.assertEqual(patched_source(input_3), original_source(input_3))\nAssertionError: Lists differ: [32368] != [32768]\n\nFirst differing element 0:\n32368\n32768\n\n- [32368]\n?    ^\n\n+ [32768]\n?    ^\n\n\n======================================================================\nFAIL: test5 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 42, in test5\n    self.assertEqual(patched_source(input_5), original_source(input_5))\nAssertionError: Lists differ: [4242538496] != [4294967296]\n\nFirst differing element 0:\n4242538496\n4294967296\n\n- [4242538496]\n+ [4294967296]\n\n======================================================================\nFAIL: test6 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 48, in test6\n    self.assertEqual(patched_source(input_6), original_source(input_6))\nAssertionError: Lists differ: [8485076992] != [8589934592]\n\nFirst differing element 0:\n8485076992\n8589934592\n\n- [8485076992]\n+ [8589934592]\n\n======================================================================\nFAIL: test7 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 54, in test7\n    self.assertEqual(patched_source(input_7), original_source(input_7))\nAssertionError: Lists differ: [4242538496] != [4294967296]\n\nFirst differing element 0:\n4242538496\n4294967296\n\n- [4242538496]\n+ [4294967296]\n\n======================================================================\nFAIL: test8 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 60, in test8\n    self.assertEqual(patched_source(input_8), original_source(input_8))\nAssertionError: Lists differ: [265158656] != [268435456]\n\nFirst differing element 0:\n265158656\n268435456\n\n- [265158656]\n+ [268435456]\n\n======================================================================\nFAIL: test9 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 66, in test9\n    self.assertEqual(patched_source(input_9), original_source(input_9))\nAssertionError: Lists differ: [8485076992] != [8589934592]\n\nFirst differing element 0:\n8485076992\n8589934592\n\n- [8485076992]\n+ [8589934592]\n\n----------------------------------------------------------------------\nRan 10 tests in 0.002s\n\nFAILED (failures=8)\n')